src/histolab/filters/image_filters.py

```diff
@@ -151,6 +164,17 @@ class Invert(ImageFilter):
     -------
     PIL.Image.Image
         Inverted image
+
+
+    Example:
+        >>> from PIL import Image
+        >>> from histolab.filters.image_filters import Invert, RgbToGrayscale
+        >>> image_rgb = Image.open("path/to/image.png") # Read an RGB image
+        >>> rgb_to_grayscale = RgbToGrayscale() # Instantiate grayscale filter
+        >>> invert = Invert() # Instantiate invert filter
+        >>> image_gray = rgb_to_grayscale(image_rgb) # Convert RGB to grayscale
+        >>> image_inv_rgb = invert(image_rgb) # Apply inversion to RGB image
+        >>> image_inv_gray = invert(image_gray) # Apply inversion to grayscale image
```

Comment:
I think `invert`  should be cap first, isn't it? Here the reason why we need `doctest` integrated

Reply:
here it is correct cause the class Invert has been already initialized above (see row 174)

Reply:
ok i see